react-components/package.json

```diff
@@ -29,21 +29,15 @@
   "devDependencies": {
```

Comment:
Should we also sort the production and dev dependencies? IMO the following dependencies should be production or peer dependencies:
* `@apollo/client`
* `braintree-web-drop-in`
* `graphql`
* `i18next`
* `informed`
* `prop-types` ?
* `react`
* `react-dom`
* `react-feather`
* `react-i18next`

Reply:
You're right, I'll take care of it.

Reply:
Would still be nice to separate the dependencies accordingly. So we know which dependencies are actually needed during "runtime" and which ones are needed during "build time" only.

I think `peerDependencies` are a bonus for when we want to define the dependencies in Venia.

Reply:
> I think peerDependencies are a bonus for when we want to define the dependencies in Venia.

They are used to signal the consumers (not only Venia) that they absolutely need those dependencies to build their app. Also, Webpack doesn't bundle `peerDependencies`, we won't be enforcing our version of (let's say) React on consumers.